### ISSUE TITLE
[Help needed] Make a link extractor undead

### ISSUE THREAD

**Issue by MrDini123** [jsergio123/script.module.resolveurl#151]
Hey man,

I am glad to see someone taking care of this awesome project still. Respect! And I am actually writing in the hope that maybe you can help me investigate.

I tried to get the waaw resolver undead which is apparently broken now. Really is a well hidden storage, they did everything just to make the parsers' life impossible.

I have modified this file:

https://github.com/jsergio123/script.module.resolveurl/blob/2cb68a26fb475cab79a009bb0ea1f0a67c229371/lib/resolveurl/plugins/waaw.py#L1-L167

And so far have it working until the get_player params extraction part:

https://gist.github.com/MrDini123/798016864763c07ddc2ecd418e9cc2bb#file-waaw-py-L67

The old regex doesn't seem to work since the site has changed.

Now data looks like this: https://gist.github.com/MrDini123/0cd0c0fdb93f9cdeacdf97bbbc410d71

I think I would have to extract the params from this:

```js
var link_m3u8 = "https://hqq.watch/player/get_md5.php?ver=2&at="+encodeURIComponent(at)+"&adb="+encodeURIComponent(adb)+"&b=1&link_1="+encodeURIComponent(faeHrZBa)+"&server_2="+encodeURIComponent(tKEzHh)+"&vid="+encodeURIComponent("allZY2grNXRGbTNuVmhWdXRXYXBRUT09")+"&ext="+encodeURIComponent(ext);
```

But how would I do that? I couldn't find a nice way with one regex.

Thanks

**Comment by Gujal00** [comment]
waaw, hqq, netu and yandexcdn are all the same site. They are using Google Recaptcha v3 on their site to pull the stream link. This can't be achieved by code yet, so for now, the resolver is broken

**Comment by Gujal00** [comment]
Fixed in #294
